var/spack/repos/builtin/packages/code-server/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class CodeServer(Package):
+    """code-server is VS Code running on a remote server,
+    accessible through the browser."""
+
+    homepage = "https://github.com/cdr/code-server"
+    url      = "https://github.com/cdr/code-server/releases/download/3.1.0/code-server-3.1.0-linux-x86_64.tar.gz"
+
+    version('3.1.1', sha256='5dd922d28b2e351c146081849d987fb1e439ee7d53b941434b2eecb2a194da71')
+    version('3.1.0', sha256='5ef85c8f280ce781a176a8b77386b333efe892755a5c325a1782e4eac6016e59')
+
+    depends_on('git@2:')
+
+    def install(self, spec, prefix):
+        ln = which('ln')
+        mkdir = which('mkdir')
+        cp = which('cp')
```

Comment:
You can remove these. Spack has builtin tools for all of these that will work regardless of OS. We plan on adding Windows support in the near future.